storage/range_raftstorage.go

```diff
@@ -348,8 +348,12 @@ func (r *Range) updateRangeInfo() error {
 		return util.Errorf("failed to lookup zone config for Range %s: %s", r, err)
 	}
 	r.SetMaxBytes(zone.RangeMaxBytes)
-	// TODO(kkaneda): Update configHashes and other fields as well (#1362)?
 
+	// Gossip configs and update configHashes if a lease is held. This is normally unnecessary
```

Comment:
Ah, I see. Thanks for taking a look! I've updated the commit to just delete the TODO comment. Will  close #1362.